packages/apptive_grid_form/lib/src/widgets/form_widget/form_content_widgets.dart

```diff
@@ -96,128 +92,94 @@ class FormDataWidget extends StatelessWidget {
-            if (properties?.defaultValue?.value != null) {
-              component.data.value = properties?.defaultValue?.value;
-            }
-            if (properties?.hidden == true) {
-              return const SizedBox();
-            }
-            final componentWidget =
-                fromModel(component, enabled: properties?.disabled != true);
-            if (componentWidget is EmptyFormWidget) {
-              // UserReference Widget should be invisible in the Form
-              // So returning without any Padding
-              return componentWidget;
-            } else {
-              return IgnorePointer(
-                ignoring: properties?.disabled == true || isSubmitting,
-                child: Padding(
-                  padding: padding,
-                  child: Builder(
-                    builder: (context) {
-                      final customBuilder =
-                          componentBuilder?.call(context, component);
-                      if (customBuilder != null) {
-                        return customBuilder;
-                      } else {
-                        return componentWidget;
-                      }
-                    },
-                  ),
-                ),
-              );
-            }
-          } else {
-            return Padding(
-              padding: padding,
-              child: Align(
-                alignment: buttonAlignment,
-                child: Builder(
-                  builder: (_) {
-                    if (isSubmitting) {
-                      return Column(
-                        mainAxisSize: MainAxisSize.min,
-                        crossAxisAlignment: CrossAxisAlignment.stretch,
-                        children: [
-                          const TextButton(
-                            onPressed: null,
-                            child: Center(
-                              child: CircularProgressIndicator.adaptive(),
-                            ),
-                          ),
-                          if (progress != null)
-                            Padding(
-                              padding: padding,
-                              child: SubmitProgressWidget(progress: progress!),
-                            ),
-                        ],
-                      );
-                    } else if (!hideButton) {
-                      return ElevatedButton(
-                        onPressed: submitForm,
-                        child: Text(
-                          buttonLabel ??
-                              data.properties?.buttonTitle ??
-                              localization.actionSend,
-                        ),
-                      );
-                    } else {
-                      return const SizedBox();
-                    }
-                  },
-                ),
+    final pages = widget.data.properties?.pageIds;
+    if (pages != null && pages.length > 1) {
+      return PageView(
```

Comment:
done, but adding the iOS swipes would be a but inconsistent, since it would only work in one direction, unless the current page is valid. I think for iOS it would be cleaner to just keep the button navigation for now